internal/controllers/import_controller.go

```diff
@@ -193,32 +188,34 @@ func (r *CAPIImportReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 }
 
 func (r *CAPIImportReconciler) reconcile(ctx context.Context, capiCluster *clusterv1.Cluster) (ctrl.Result, error) {
+	log := log.FromContext(ctx)
+
 	// fetch the rancher clusters
-	rancherClusterHandler := rancher.NewClusterHandler(ctx, r.RancherClient)
-	rancherClusterName := turtelesnaming.Name(capiCluster.Name).ToRancherName()
+	rancherCluster := &provisioningv1.Cluster{ObjectMeta: metav1.ObjectMeta{
+		Namespace: capiCluster.Namespace,
+		Name:      turtelesnaming.Name(capiCluster.Name).ToRancherName(),
```

Comment:
this needs to be fixed for the import of package itself as well:

```suggestion
		Name:      turtlesnaming.Name(capiCluster.Name).ToRancherName(),
```

Reply:
Looking at this line, I really asking myself why I did it like this 😔